_data/DeepanshuJain/DeepanshuJain.md

```diff
@@ -0,0 +1,5 @@
+name: Deepanshu Jain
+institution: DIT University 🚩
+profile_pic: DeepanshuJain.png
```

Comment:
Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `DeepanshuJain.png` to `DeepanshuJain.PNG ` in the markdown file.
2. Rename `DeepanshuJain.PNG ` to `DeepanshuJain.png`.
I'd suggest you rename the file!